stock_picking_batch_extended/models/stock_batch_picking.py

```diff
@@ -99,6 +105,22 @@ def action_print_picking(self):
     def remove_undone_pickings(self):
         """Remove of this batch all pickings which state is not done / cancel."""
         self.mapped("active_picking_ids").write({"batch_id": False})
+        self.verify_state()
+
+    def verify_state(self):
```

Comment:
Last but not least, this is an addition in behavior from previous version.

IMHO, this module deserves a good DESCRIPTION.rst in readme folder to explain what it does as 'extended' is too general term to understand at first sight. Thanks

Reply:
I fix that. I

Reply:
Ok, nevertheless, a good module description could be a great improvement :smiley: